static/app/views/performance/newTraceDetails/traceRenderers/virtualizedViewManager.tsx

```diff
@@ -127,6 +127,20 @@ export class VirtualizedViewManager {
   private readonly ROW_PADDING_PX = 16;
   private scrollbar_width: number = 0;
 
+  replay_indicators: {
+    current: {
+      ref: HTMLElement | null;
+      timestamp: number | undefined;
+    };
+    hover: {
+      ref: HTMLElement | null;
+      timestamp: number | undefined;
+    };
+  } = {
+    current: {ref: null, timestamp: undefined},
+    hover: {ref: null, timestamp: undefined},
+  };
```

Comment:
```suggestion
  type VerticalIndicator = {
    timestamp: number | undefined,
    ref: HTMLElement | null
  }
  replay_indicators: {
    current: VerticalIndicator,
    hover: VerticalIndicator
    } = {
    current: {ref: null, timestamp: undefined},
    hover: {ref: null, timestamp: undefined},
  };
```

Reply:
A more generic API leaves you more flexibility to add different indicators in the future